backend/database/seed.go

```diff
@@ -60,4 +60,65 @@ func SeedTierlists() error {
 
 	fmt.Println("Tierlists Seeded Successfully")
 	return nil
+}
+
+func SeedTiers() error {
+	if DB == nil {
+		return fmt.Errorf("database connection unavailable")
+	}
+
+	var tiersResult []models.Tiers
+	result := DB.Find(&tiersResult)
+
+	if result.RowsAffected > 0 {
+		return nil
+	}
+
+	tiers := []models.Tiers{
+		{TierlistID: 1, Text: "S", Colour: "f4ee1d"},
+		{TierlistID: 1, Text: "A", Colour: "d81621"},
+		{TierlistID: 1, Text: "B", Colour: "00bef1"},
+		{TierlistID: 1, Text: "C", Colour: "37f100"},
+		{TierlistID: 1, Text: "D", Colour: "ffffff"},
+		{TierlistID: 1, Text: "F", Colour: "000000"},
+	}
+
+	for _, tier := range tiers {
+		if err := DB.Create(&tier).Error; err != nil {
+			return fmt.Errorf("failed to seed tiers")
+		}
+	}
+
+	fmt.Println("Tiers Seeded Successfully")
+	return nil
+}
+
+func SeedItems() error {
+	if DB == nil {
+		return fmt.Errorf("database connection unavailable")
+	}
+
+	var itemsResult []models.Items
+	result := DB.Find(&itemsResult)
+
+	if result.RowsAffected > 0 {
+		return nil
+	}
+
+	items := []models.Items{
+		{TierlistID: 1, Text: "Nacht der Untoten", Image: "https://static.wikia.nocookie.net/callofduty/images/2/2b/Nacht_Der_Untoten_Menu_Selection_WaW.png/revision/latest?cb=20161009103531", TierID: 4},
+		{TierlistID: 1, Text: "Verrückt", Image: "https://static.wikia.nocookie.net/callofduty/images/a/a1/Verruckt_Menu_Selection_WaW.png/revision/latest?cb=20161009103542", TierID: 3},
+		{TierlistID: 1, Text: "Shi No Numa", Image: "https://static.wikia.nocookie.net/callofduty/images/2/2f/Shi_No_Numa_Menu_Selection_WaW.png/revision/latest?cb=20161009103553", TierID: 2},
+		{TierlistID: 1, Text: "Der Riese", Image: "https://static.wikia.nocookie.net/callofduty/images/8/86/Der_Riese_Menu_Selection_WaW.png/revision/latest?cb=20161009103603", TierID: 1},
+		{TierlistID: 1, Text: "Moon", Image: "https://static.wikia.nocookie.net/callofduty/images/c/cc/Moon_Menu_Selection_BO.png/revision/latest?cb=20240710075602", TierID: 1},
+	}
+
+	for _, item := range items {
+		if err := DB.Create(&item).Error; err != nil {
+			return fmt.Errorf("failed to seed items")
+		}
+	}
+
+	fmt.Println("Items Seeded Successfully")
+	return nil
```

Comment:
Seed Functions are looking good, just see the comment in `seed.go` for simplifying it and combining into a single function.